docs/tutorial/part-two/index.md

```diff
@@ -72,26 +72,26 @@ You should now have a structure like this:
 │       └── global.css
 ```
 
-2. Define some styles in the `global.css` file:
+2. `Global.css` फ़ाइल में कुछ सीएसएस स्टाइल्स को परिभाषित करें:
 
 ```css:title=src/styles/global.css
 html {
   background-color: lavenderblush;
 }
 ```
 
-> Note: The placement of the example css file in a `/src/styles/` folder is arbitrary.
+> नोट: उदाहरण लिए सीएसएस फ़ाइल का प्लेसमेंट `/src/styles/` फ़ोल्डर में मनमाना है।
 
-#### ✋ Include the stylesheet in `gatsby-browser.js`
+#### ✋ `Gatsby-browser.js` में स्टाइलशीट को शामिल करें
 
-1. Create the `gatsby-browser.js`
+1. एक नयी फाइल `Gatsby-browser.js` बनाएँ
 
 ```shell
 cd ../..
 touch gatsby-browser.js
 ```
 
-Your project's file structure should now look like this:
+आपके प्रोजेक्ट की फ़ाइल स्ट्रक्चर अब इस तरह दिखनी चाहिए:
```

Comment:
`की` => `का`

`दिखनी` => `दिखना`

This also, not very sure, what you think?

Reply:
```प्रोजेक्ट की फ़ाइल`` sounds better than ```प्रोजेक्ट का फ़ाइल```
hence
```आपके प्रोजेक्ट की फ़ाइल स्ट्रक्चर अब इस तरह दिखनी चाहिए``` makes more sense

Reply:
here `की` is referring to `स्ट्रक्चर` not `फ़ाइल` so I think `आपके प्रोजेक्ट का फ़ाइल स्ट्रक्चर अब इस तरह दिखना चाहिए`

If you read the whole sentence this will make more sense.

Reply:
ok